packages/fxa-settings/src/pages/Signin/SigninTokenCode/container.test.tsx

```diff
@@ -102,17 +103,20 @@ function resetMockSensitiveDataClient() {
     keyFetchToken: MOCK_KEY_FETCH_TOKEN,
     unwrapBKey: MOCK_UNWRAP_BKEY,
   });
+  mockSensitiveDataClient.KeyStretchUpgradeData = undefined;
 }
 
-async function render() {
+async function render(mocks: Array<MockedResponse>) {
   renderWithLocalizationProvider(
-    <LocationProvider>
-      <SigninTokenCodeContainer
-        {...{
-          integration,
-        }}
-      />
-    </LocationProvider>
+    <MockedProvider mocks={mocks} addTypename={false}>
```

Comment:
Is this change necessary if we don't pass in mocks in any of the render calls?

Reply:
It appears to be. I was getting an error without it. When the control renders, the 'useMutation' functions get called, and that errors if there's MockedProvider wrapping the component.

Reply:
In that case I'm surprised it wasn't needed for signin_totp_code

Reply:
Just took a second look at this, it's because of a mock in `SignInTokenCode` on  `jest.spyOn(ApolloModule, 'useMutation')`. I feel like the more consistent way to do this is using the `MockedProvider`.